scripts/travis/.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: true
+sudo: false
```

Comment:
Need to update line 6. Also, is this sudo change necessary?

Reply:
Line 6 of this file also needs to be 5.6.

Reply:
Fixed! I also added a doc comment to advise the user this should match whatever version their prod environment is in.